kyo-prelude/shared/src/test/scala/kyo2/kernel/BoundaryTest.scala

```diff
@@ -115,7 +115,7 @@ class BoundaryTest extends Test:
     }
 
     "detects for non-context effects" in {
-        assertDoesNotCompile("summon[Boundary[NotContextEffect, Any]]")
+        assertDoesNotCompile("Boundary[NotContextEffect, Any]")
     }
```

Comment:
Check the compiler error here - you may need to pass a value. You could pass a parameter if you don't want to define an actual effect.

Reply:
It looks ok:
```
[error] 118 |        Boundary[NotContextEffect, Any]
[error]     |                                       ^
[error]     | Expected context effects. Found: 'BoundaryTest.this.NotContextEffect'
```